src/mijnamsterdam.d.ts

```diff
@@ -66,3 +66,14 @@ declare const MA_BUILD_ID: string;
 
 // Is the app built on azure or not.
 declare const MA_IS_AZ: string;
```

Comment:
*Waarschijnlijk even buiten scope.* Deze valt me gewoon op ineens. Niet beter als boolean? Ik zie dat die uiteindelijk ook als boolean gebruikt word en alleen '{true, false, undefined}' er op word gedaan. Misschien gewoon beter meteen in `defineConfig` casten bij het uitlezen? Als we dan `getFromEnv` kunnen gebruiken hoeven we ook geen rekening meer te houden met undefined.

Reply:
![Screenshot 2024-09-12 at 09 23 09](https://github.com/user-attachments/assets/00231e47-3af4-4558-9689-3fdcbdfb4cc4)

Reply:
Het is configuratie voor een build step van Vite. En worden als token "string" replaced tijdens bouwen. Mogelijk kan hier wel een boolean staan idd. Maar IS_AZ is niet meer nodig dus haal ik weg.

In de FE is getFromEnv sowieso niet mogelijk omdat er in een FE geen env bestaat. We gebruiken voor process.env import.meta in de fe. Tijdens build worden alle referenties van import.meta.X vervangen met process.env.X door Vite.

Reply:
Ah duidelijk bedankt voor de uitleg